daffodil-test/src/test/scala/org/apache/daffodil/section05/simple_types/TestSimpleTypes.scala

```diff
@@ -189,7 +189,24 @@ class TestSimpleTypes {
   @Test def test_dateTimeBin17() { runner.runOneTest("dateTimeBin17") }
   @Test def test_dateTimeBin18() { runner.runOneTest("dateTimeBin18") }
   @Test def test_dateTimeBin19() { runner.runOneTest("dateTimeBin19") }
-  @Test def test_dateBin2() { runner.runOneTest("dateBin2") }
+  @Test def test_dateTimeBin20() { runner.runOneTest("dateTimeBin20") }
```

Comment:
I'm not sure what you mean. The only test in debug is dateTimeBinBCD3, and that one is commented out in here. I planned to just fix the test based on discussion in the review so there won't be any tests added to debug. 

I did change the name of test_dateBin2 to test_dateBinInvalid, because the original name wasn't really following a sequence and didn't make sense. Maybe that's where the confusion comes in.